about.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="style.css">
+        <title>Capstone</title>
+    </head>
+    <body class="bodyart">
+        <header id="header" class="header">
+            <nav class="navbar">
+                <ul class="nav-list">
+                    <li>
+                        <img src="./images/Rectangle.png" class="facebook" alt="facebook"/>
+                    </li>
+                    <li>
+                        <img src="./images/twitter.png" class="facebook" alt="facebook"/>
+                    </li>
+                    <li>
+                        <a class="header-styles" href="#">English</a>
+                    </li>
+                    <li>
+                        <a class="header-styles" href="#">My Page</a>
+                    </li>
+                    <li>
+                        <a class="header-styles" href="#">Logout</a>
+                    </li>
+                    <li class="whitespace">                                      
+                    </li>
+                </ul>
+            </nav>
+        </header>
+        <section class="section1">
```

Comment:
- [ ] As suggested by the previous reviewer, the parts of your code that he highlighted should be moved to your about page. Please kindly watch this [video](https://watch.screencastify.com/v/UgXfOlXJAOKJFbwzGKif) by the previous reviewer to understand more.

Reply:
hi
I don't understand your answer.
As suggested by the previous reviewer, the parts of your code that h
![2023_07_08_21_10_52_Window](https://github.com/jovasal/Capstone/assets/35693330/d8d03b62-640a-41c8-8d1c-5bd44917d6ba)
e highly
![2023_07_08_21_10_06_HTML_CSS_JavaScript_capstone_project_Conference_page_by_jovasal_Pull_Reque](https://github.com/jovasal/Capstone/assets/35693330/04ab26b6-3f47-4445-bb5f-69e833382297)
ghted should be moved to your About page. Please kindly watch this [video](https://watch.screencastify.com/v/UgXfOlXJAOKJFbwzGKif) by the previous reviewer to understand more.
I create an About page. The About page is not the design, I create another page for this purpose and I explain it in my pull request.
Please, recheck it

If you check the design. About is my creation, and is not in design. as @AbayomiOlaoye said. he believes, but I'm sure that is not in that part and I explain that.

I think that you made a mistake and need to chek